bin/GameIndex.dbf

```diff
@@ -139,10 +139,10 @@ Name   = Playstation 2 - Demo Disc 2001
 Region = PAL-M5
 [patches]
 	// 7ACF7E03
-	comment=You must enable the EETimingHack when playing the WRC demo to avoids random hangs.
-	comment=You must change the EE Clamping mode to Full when playing the Klonoa 2 demo to avoids misplaced objects.
-	comment=You must change the VU Clamping mode to extra when playing the Klonoa 2 demo to fixes reflections issues.
 	comment=You must use the ZeroSPU2 plugin when playing the Klonoa 2 demo to avoids bad/missing sounds.
+EETimingHack = 1 // Fixes WRC demo hanging when the car cross the starting line.
+eeClampMode = 3 // In the Klonoa 2 demo objects appear in wrong places without it.
+vuClampMode = 2 // Fixes water reflection in the Klonoa 2 demo.
```

Comment:
I don't think it's a good idea to add these for demo discs, other demo games might break.

Reply:
I tested each of them before doing this and none of them actualy break with these fixes.
But it's up to you, do you want I remove them?

Reply:
Yes change them back.

Reply:
Done.